mpt/impl/imap-mailbox/core/src/main/java/org/apache/james/mpt/imapmailbox/suite/ACLScriptedTestProtocol.java

```diff
@@ -0,0 +1,82 @@
+package org.apache.james.mpt.imapmailbox.suite;
+
+import java.util.Locale;
+
+import org.apache.james.mailbox.model.MailboxACL;
+import org.apache.james.mailbox.model.MailboxACL.MailboxACLRights;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mpt.api.HostSystem;
+import org.apache.james.mpt.api.ImapHostSystem;
+import org.apache.james.mpt.imapmailbox.GrantRightsOnHost;
+import org.apache.james.mpt.imapmailbox.MailboxMessageAppender;
+import org.apache.james.mpt.script.ImapScriptedTestProtocol;
+import org.apache.james.mpt.script.SimpleScriptedTestProtocol;
+
+public class ACLScriptedTestProtocol extends ImapScriptedTestProtocol {
```

Comment:
As explained before : I don't agree, we can rely on the type system, it's much better than runtime exceptions.